src/main/kotlin/com/cognifide/gradle/aem/common/instance/LocalInstance.kt

```diff
@@ -21,19 +23,34 @@ class LocalInstance private constructor(aem: AemExtension) : AbstractInstance(ae
     override lateinit var password: String
 
     var debugPort: Int = 5005
+    var debugAddress: String = ""
+    private val debugAddressWithPort: String
+        get() = when (debugAddress) {
+            "*" -> "0.0.0.0:$debugPort"
+            "" -> "$debugPort"
+            else -> "$debugAddress:$debugPort"
+        }
 
     @get:JsonIgnore
     val jvmOptsDefaults: List<String>
         get() = mutableListOf<String>().apply {
-            if (debugPort > 0) {
-                add("-Xdebug")
-                add("-Xrunjdwp:transport=dt_socket,address=$debugPort,server=y,suspend=n")
+            if (debugPort in 1..65535) {
```

Comment:
Why you add such constraint? 😉 I am not sure if it gives any value

Reply:
Theoritically you are right but in practice...

Reply:
In case you make a mistake in port and type higher than that, instance won't start at all without a message. This is why I added this constraint.